README.md

```diff
@@ -119,21 +119,23 @@ struct Euro : twig::stronk<Euro, double, twig::unit>
 
 void introducing_another_type()
 {
-    // There are make functions for std::ratio like types. It simply scales the input value and does not change the type
-    WattHours watt_hours = twig::make<std::mega, WattHours>(1.);
+    // twig::make allows you to scale the input value but it does not change the resulting type
+    WattHours one_mega_watt_hour = twig::make<std::mega, WattHours>(1.);
     // Now we can generate a new type which consists of 3 types: `Euro / (Watt * Hours)`
-    auto euros_per_watt_hour = Euro {300.} / watt_hours;
+    auto euros_per_mega_watt_hour = Euro {300.} / one_mega_watt_hour;
 
-    // This flexibility allows us to very expessively write code while having the type system checking our code.
-    Euro price_for_buying_5_megawatt = euros_per_watt_hour * twig::make<std::mega, WattHours>(5.);
+    // This flexibility allows us to write expessive code, while having the type system check our implementation.
+    Euro price_for_buying_5_mega_watt_hours = euros_per_mega_watt_hour * twig::make<std::mega, WattHours>(5.);
 
-    auto watt_hours_per_euro = 1. / euros_per_watt_hour;  // `(Watt * Hours) / Euro`
-    WattHours watt_hours_affordable_for_500_euros = watt_hours_per_euro * Euro {500.};
+    auto mega_watt_hours_per_euro = 1. / euros_per_mega_watt_hour;  // `(Watt * Hours) / Euro`
+    WattHours mega_watt_hours_affordable_for_500_euros = mega_watt_hours_per_euro * Euro {500.};
 }
 ```
 
 Units are a great way of using the type system to validate your code.
 
+Credit to https://www.foonathan.net/2016/10/strong-typedefs/ for a great amount of inspiration
```

Comment:
You could also consider linking his GitHub repo: https://github.com/foonathan/type_safe. You could also put this under why, and write how stronk supplements his library